PlayerMethods.h

```diff
@@ -2434,7 +2433,7 @@ namespace LuaPlayer
 #ifndef AZEROTHCORE
             player->UnbindInstance(map, (Difficulty)difficulty);
 #else
-            sInstanceSaveMgr->PlayerUnbindInstance(player->GetGUIDLow(), map, Difficulty(difficulty), true, player);
+            sInstanceSaveMgr->PlayerUnbindInstance(player->GetGUID(), map, Difficulty(difficulty), true, player);
```

Comment:
potentially same here

Reply:
This line is specific to AzerothCore only and it's okay.